react-frontend/src/App.js

```diff
@@ -0,0 +1,49 @@
+import './App.css'
+import Faucet from './components/Faucet'
+import Web3 from 'web3'
+import faucetABI from './abi/SwanFaucet.json'
+import tokenABI from './abi/USDC.json'
+
+function App() {
```

Comment:
I think you can remove this file, since we are not calling contract through web

Reply:
I will be using two call methods that do not require signer